docs/ROADMAP.md

```diff
@@ -0,0 +1,15 @@
+# Champ - Roadmap
+
+This document describes the current status and the upcoming milestones of the Champ project.
+
+_Updated: 2024-03-27T07:42:51.399Z_
+
+## Champ
+
+#### Milestone Summary
+
+| Status | Milestone                                                                                  | Goals |       ETA       |
+| :----: | :----------------------------------------------------------------------------------------- | :---: | :-------------: |
+|   ✅   | **[Model and test data on Huggingface](https://huggingface.co/fudan-generative-ai/champ)** | 2 / 2 | Tue Mar 26 2024 |
+|   🚀   | **[Data preprocessing code release]()**                                                    | 0 / 1 | Thu Mar 28 2024 |
+|   🚀   | **[Trainging code release]()**                                                             | 0 / 1 | Mon Apr 01 2024 |
```

Comment:
Add the milestone of releasing training dataset

Reply:
done